dev-util/bear/bear-2.4.2.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Automatically generate compilation database for Clang tooling"
+DESCRIPTION="Bear records the flags passed to the compiler for each translation unit and \
+stores them in a JSON file. This file can be used by Clang's tooling interface \
+and programs like clang-check to process a translation unit.\
+\
+cmake supports the generation of JSON compilation databases out of the box. \
+For any other build system that does not support this, Bear can be used \
+instead to intercept the invocation of the compiler."
+HOMEPAGE="https://github.com/rizsotto/Bear"
+COPYRIGHT="2012-2019 by László Nagy"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/rizsotto/Bear/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="e80c0d622a8192a1ec0c0efa139e5767c6c4b1defe1c75fc99cf680c6d1816c0"
+SOURCE_DIR="Bear-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64 ?arm"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	bear$secondaryArchSuffix = $portVersion compat >= 3
```

Comment:
usually no "compat" on package provides, only commands or libs.

Reply:
Thanks man, I removed the compat.